build_docs.pl

```diff
@@ -680,32 +673,6 @@ sub init_env {
         or die "Please install <xsltproc>";
 }
 
-#===================================
```

Comment:
I moved this so it'd be by its buddies.

Reply:
And I sorted it.

Reply:
:+1: Command-line options should stick together in these troubled times.